config/settings.yml

```diff
@@ -869,6 +869,10 @@
   :secret_filter: []
 :messaging:
   :type: kafka
+:miq_request:
+  :history:
+    :purge_window_size: 500
+    :keep_miq_requests: 6.months
```

Comment:
I picked 500.  I think it's a small enough table that grows based on user input so smaller batches makes sense to keep the list of ids smaller.  Also, request_logs can be a lot per request, so it would be good to ensure we can get smaller batches completed before timing out, so less is rolled back on a timeout.